### PR TITLE
feat: Migrate to Jetpack (AndroidX)

### PR DESCRIPTION
**Pull request by galuszkak**

### CODE REVIEW

**Comment by galuszkak** [comment]
Hey @tbruyelle ,

Thanks for awesome project.

It looks like tests are passing on Travis: https://travis-ci.org/github/tbruyelle/RxPermissions/builds/702404691

Not sure why aren't reporting here in this PR. I would be grateful for review and eventually merging this into this project.

Thanks!

**Comment by tbruyelle** [comment]
Hello @galuszkak 

Sorry I can't review your change, I'm not aware of androidX since I'm no longer an android developer. 
Maybe @epool can help on this ?

File: lib/build.gradle [hunk 1]
```diff
@@ -14,7 +18,7 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
-
+    testOptions.unitTests.includeAndroidResources = true
```
Comment (epool) [inline lib/build.gradle:21]: Why this change is required?

Comment (galuszkak) [reply lib/build.gradle:21]: Hi @epool ,

There were warning in test output about legacy resource mode that is apparently deprecated.:
```
[Robolectric] NOTICE: legacy resources mode is deprecated; see http://robolectric.org/migrating/#migrating-to-40
```

After adding this option output from tests is much more clean.

Does this answer Your question? 

Thanks!

**Comment by tbruyelle** [comment]
thx @epool

**Comment by tbruyelle** [comment]
I've published v0.11 to embed this change.
